README.md

```diff
@@ -1,39 +1,126 @@
-![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/Azure.azure-iot-sdk-python)
-
 # Azure IoT Hub Python SDKs v2 - PREVIEW
 
-This repository contains the code for the future v2.0.0 of the Azure IoT SDKs for Python. The goal of v2.0.0 is to be a complete rewrite of the existing SDK that maximizes the use of the Python language and its standard features rather than wrap over the C SDK, like v1.x.x of the SDK did.
+![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/Azure.azure-iot-sdk-python)
+
+This repository contains code for the Azure IoT SDKs for Python.  This enables python developers to easily create IoT devices that work semealessly 
 
 *If you're looking for the v1.x.x client library, it is now preserved in the [v1-deprecated](https://github.com/Azure/azure-iot-sdk-python/tree/v1-deprecated) branch.*
 
 **Note that these SDKs are currently in preview, and are subject to change.**
 
-# SDKs
+## Python SDK
 
-This repository contains the following SDKs:
+This repository contains the following:
 
-* [Azure IoT Device SDK](azure-iot-device) - /azure-iot-device
-    * Provision a device using the Device Provisioning Service for use with the Azure IoT hub
-    * Send/receive telemetry between a device or module and the Azure IoT hub or Azure IoT Edge device
-    * Handle direct methods invoked by the Azure IoT hub on a device
-    * Handle twin events and report twin updates
-    * *Still in development*
-        - *Blob/File upload*
-        - *Invoking method from a module client onto a leaf device*
+* *Azure IoT Device SDK*
 
-* Azure IoT Hub SDK **(COMING SOON)**
-    * Do service/management operations on the Azure IoT Hub
+  * Send/receive telemetry between a device or module and the Azure IoT hub or Azure IoT Edge device
+  * Handle direct methods invoked by the Azure IoT hub on a device
+  * Handle twin events and report twin updates
+  * Provision a device using the Device Provisioning Service for use with the Azure IoT hub
+  * *Still in development*
+    * *Blob/File upload*
+    * *Invoking method from a module client onto a leaf device*
 
-* Azure IoT Hub Provisioning SDK **(COMING SOON)**
-    * Do service/management operations on the Azure IoT Device Provisioning Service
+* Azure IoT Hub SDK
+  * Do service/management operations on the Azure IoT Hub
+  * *Still in development*
+    * C2D messaging
+    * Direct Methods
+    * Configurations
+    * Query devices
+    * IoTHub Job
 
-# How to install the SDKs
+  * Do service/management operations on the Azure IoT Device Provisioning Service
 
-```
+## How to install the SDKs
+
+The SDK is contained in various Pip Packages that can be installed with the following commands:
+
+For the Device SDK the following command:
+
+```Shell
 pip install azure-iot-device
 ```
 
-# Contributing
+For the IoTHub SDK the following command:
+
+```Shell
+pip install azure-iot-hub
+```
+
+## Python SDKs Feature
```

Comment:
This does raise the question though of if we really want the full table feature chart in the monorepo root - I think it's worth considering if that information is better suited to the library-specific READMEs, and keep the monorepo root README as simple as possible, just directing people to the correct documentation.

In the short term I think it's fine though.